### DISCUSSION TITLE
記事フィード内の記事において、コメント数をベースにした記事のランキング機能を作成してほしい

### DISCUSSION THREAD

**Discussion by daiki0730** [increments/qiita-discussions#84] [要望・アイデア・不具合]
# 概要
記事フィード内の記事において、
コメント数をベースにした記事のランキング機能を作成希望
(トレンドはLGTM数をベースに算出しているという認識です)

# 理由
コメント(エンジニアのディスカッション)にも学びや価値があると考えているためです

**Comment by kskwtnk** [comment]
@daiki0730 

ご意見ありがとうございます！

結論としてはコメント数**のみ**を基準にしたランキングを作ることは無いと思われます 🙏 

ただ、コメントも価値があるのは間違い無くそうです！
記事を投稿する→コメントでより良い書き方の指摘・アドバイスがある→記事がアップデートされてより良いものになる、といった流れはときどき見かけます

現状トレンドに乗る乗らないの判定はLGTM数のみなので、近いことを実現するとしたらコメントの数もトレンドの計算に入れる、となるかと思います！
